app/components/container/containerController.js

```diff
@@ -197,6 +197,104 @@ function ($scope, $state, $stateParams, $filter, Container, ContainerCommit, Con
     });
   };
 
+  $scope.recreate = function() {
+    var config = $scope.container.Config;
+    // HostConfig
+    config.HostConfig = $scope.container.HostConfig;
+    // Name
+    config.name = $scope.container.Name.replace(/^\//g, '');
+    // Network
+    var mode = config.HostConfig.NetworkMode;
+    // TODO get ip addrs
```

Comment:
Just test it on ipv4 addr, it's OK
So I remove TODO https://github.com/portainer/portainer/pull/855/commits/383e70e22754e8a245184e0590396fcb6ef738f8